client/components/git-readme.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+import React from 'react'
+
+export const GitReadme = React.createClass({
+  propTypes: {
+    github: React.PropTypes.object.isRequired
+  },
+  render () {
+    return (
+      <div className="panel-module" dangerouslySetInnerHTML={{__html: this.props.github.readme}} />
```

Comment:
What are you thoughts on using `dangerouslySetInnerHTML={{__html: this.props.github.readme}}` versus rendering inside a `<p>{this.props.github.readme} </p>`